apps/payment/app.go

```diff
@@ -20,16 +20,15 @@ import (
 
 	"perun.network/go-perun/channel"
 	"perun.network/go-perun/log"
-	"perun.network/go-perun/wallet"
 )
 
 // App is a payment app.
 type App struct {
-	Addr wallet.Address
+	Addr channel.AppID
```

Comment:
I think it would make more sense to then also rename it from `Addr` to ID`.

Reply:
Done